Cargo.toml

```diff
@@ -25,13 +25,13 @@ name = "gdk_pixbuf"
 v2_32 = ["gdk-pixbuf-sys/v2_32"]
 v2_36 = ["v2_32", "gdk-pixbuf-sys/v2_36"]
 v2_36_8 = ["v2_36", "gdk-pixbuf-sys/v2_36_8"]
-dox = ["glib/dox", "gdk-pixbuf-sys/dox"]
+dox = ["glib/dox", "gdk-pixbuf-sys/dox", "futures"]
```

Comment:
Maybe combine this with removing of the `futures` feature here?

Reply:
Oh brainfart. We remove the `subclass` feature, not the `futures` feature. Ignore this part :)

Reply:
Ignored then. :)